### PR TITLE
add ashhuahua

### PR DESCRIPTION
**Pull request by DBcosmos**
add ashhuahua (burn derivative)

### CODE REVIEW

File: images/chihuahua/factory/DBcosmos/chihuahua1hplyuj2hzxd75q8686g9vm3uzrrny9ggvt8aza2csupgdp98vg2sp0e3h0/ashhuahua.png
Comment (editaahn) [inline images/chihuahua/factory/DBcosmos/chihuahua1hplyuj2hzxd75q8686g9vm3uzrrny9ggvt8aza2csupgdp98vg2sp0e3h0/ashhuahua.png:1]: The image directory and file name has to be same as `coinImageUrl` that you wrote in JSON.

**Comment by DBcosmos** [comment]
@editaahn directories/links updated :)

**Comment by editaahn** [comment]
Hi! It still doesn't follow the rules for registration. You could find it out by validating the code by yourself.
image

**Comment by DBcosmos** [comment]
I sorry, I'm not a dev will give this ago. Could it be a problem with the capitalisation of the ashHUAHUA in the json but the png file is not capitalised (ashhuahua)? if so that is fixed :)

**Review by editaahn — COMMENTED** [review]
- The coin image directory should be same as coinMinimalDenom. `factory/chihuahua1hplyuj2hzxd75q8686g9vm3uzrrny9ggvt8aza2csupgdp98vg2sp0e3h0/uhuahua.ash`
- ampGASH image size needs to be changed as 256*256px.

**Pull request merged**